### PR TITLE
fix(ContactRequests): Fix `sensor.onClicked` to `onClicked`

### PR DESCRIPTION
**Pull request by borismelnik**
### What does the PR do

Fix signal which doesn't work anymore

### Affected areas

ContactRequests

### Screenshot of functionality (including design for comparison)

- [ ] I've checked the design and this PR matches it

### CODE REVIEW

File: ui/app/AppLayouts/Profile/views/MessagingView.qml [hunk 1]
```diff
@@ -216,7 +216,7 @@ SettingsContentBase {
             Layout.fillWidth: true
             title: qsTr("Contacts, Requests, and Blocked Users")
             requestsCount: root.contactsStore.receivedContactRequestsModel.count
-            sensor.onClicked: Global.changeAppSectionBySectionType(Constants.appSection.profile,
+            onClicked: Global.changeAppSectionBySectionType(Constants.appSection.profile,
```
Comment (micieslak) [inline ui/app/AppLayouts/Profile/views/MessagingView.qml:219]: little thing: formatting of the line below could be fixed :)

**Review by micieslak — APPROVED** [review]
lgtm

File: ui/app/AppLayouts/Profile/views/MessagingView.qml (hunk 1 above)
Comment (borismelnik) [reply ui/app/AppLayouts/Profile/views/MessagingView.qml:219]: It some kind of `hot fix` I don't want to fix something else in this PR :)

**Review by jrainville — APPROVED** [review]

**Review by anastasiyaig — APPROVED** [review]

**Comment by anastasiyaig** [comment]
@borismelnik next time please log a simple issue and link it to the PR, so QA team will see it on the board (we dont see PRs, we see only issues)

**Pull request merged**